src/soundbase.cpp

```diff
@@ -167,20 +167,18 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ).arg ( strSystemDriverTechniqueName ) +
-                                    "</b><br><br>" + tr ( "These are all the available drivers with error messages:" ) + "<ul>";
+            QString sErrorMessage = tr ( "<b>No usable %1 audio device found.</b><br><br>" ).arg ( strSystemDriverTechniqueName );
```

Comment:
I don't suppose it's really a big issue, but if this reads _"No usable Jamulus audio device found."_ then will some people think this is due to them not having something called a "Jamulus audio device" and go searching the docs for how to install one? 

Would it be slightly safer to just say "_Jamulus couldn't find a usable audio device._"? (also active voice so better)

Reply:
%1 shouldn’t ever say Jamulus. So I think it‘s fine

Reply:
@gilgongo Any news here? I wouldn't say %1 could ever be Jamulus

Reply:
Ok, but would it be clearer/less ambiguous so say "Jamulus couldn't find a usable %1 audio device"? (would be in active voice too)

Reply:
Ok. Done